backend/tests/unit/apps/github/api/internal/dataloaders/pull_request_test.py

```
@@ -129,15 +131,102 @@ async def test_missing_project_returns_empty_list(self, mock_pull_request, mock_
+
+        mock_pull_request.objects.filter.return_value.annotate.return_value.filter.assert_called_once_with(
+            row_number__gt=4, row_number__lte=8
+        )
+
+    @patch(
+        "apps.github.api.internal.dataloaders.pull_request.get_results_by_keys",
+        new_callable=AsyncMock,
+    )
+    @patch("apps.github.api.internal.dataloaders.pull_request.PullRequest")
+    @pytest.mark.asyncio
+    async def test_maps_pull_requests_to_issue_ids(self, mock_pull_request, mock_get_results):
```

Ruff (0.16.1)

[warning] Unused method argument: `mock_pull_request`

(ARG002)